examples/notebooks/create_deployment.ipynb

```diff
@@ -12,48 +23,288 @@
+       "                                                'termination_message_policy': None,\n",
+       "                                                'tty': None,\n",
+       "                                                'volume_mounts': None,\n",
+       "                                                'working_dir': None}],\n",
+       "                                'dns_policy': 'ClusterFirst',\n",
+       "                                'host_ipc': None,\n",
+       "                                'host_network': None,\n",
+       "                                'host_pid': None,\n",
+       "                                'hostname': None,\n",
+       "                                'image_pull_secrets': None,\n",
+       "                                'init_containers': None,\n",
+       "                                'node_name': None,\n",
+       "                                'node_selector': None,\n",
+       "                                'restart_policy': 'Always',\n",
+       "                                'scheduler_name': None,\n",
+       "                                'security_context': {'fs_group': None,\n",
+       "                                                     'run_as_non_root': None,\n",
+       "                                                     'run_as_user': None,\n",
+       "                                                     'se_linux_options': None,\n",
+       "                                                     'supplemental_groups': None},\n",
+       "                                'service_account': None,\n",
+       "                                'service_account_name': None,\n",
+       "                                'subdomain': None,\n",
+       "                                'termination_grace_period_seconds': 30,\n",
+       "                                'tolerations': None,\n",
+       "                                'volumes': None}}},\n",
+       " 'status': {'available_replicas': None,\n",
+       "            'conditions': None,\n",
+       "            'observed_generation': None,\n",
+       "            'ready_replicas': None,\n",
+       "            'replicas': None,\n",
+       "            'unavailable_replicas': None,\n",
+       "            'updated_replicas': None}}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "extensions.create_namespaced_deployment(namespace='default', body=deployment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete Deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#extensions.delete_namespaced_deployment(name='nginx-deployment', namespace='default', body=client.V1DeleteOptions(),grace_period_seconds = 56, propagation_policy =  \"Background\")"
```

Comment:
Thanks to @vascofg, updating my version of Kubernetes solved my cascading deletion issue.